e2e/cloudfront/cloudfront_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cloudfront_test
+
+import (
+	"net/http"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/aws/copilot-cli/e2e/internal/client"
+)
+
+var _ = Describe("CloudFront", func() {
+	const domainName = "copilot-e2e-tests.ecs.aws.dev"
+
+	Context("when creating a new app", func() {
+		var appInitErr error
+
+		BeforeAll(func() {
+			_, appInitErr = cli.AppInit(&client.AppInitRequest{
+				AppName: appName,
+				Domain:  domainName,
+			})
+		})
+
+		It("app init succeeds", func() {
+			Expect(appInitErr).NotTo(HaveOccurred())
+		})
+
+		It("app init creates a copilot directory", func() {
+			Expect("./copilot").Should(BeADirectory())
+		})
+
+		It("app ls includes new application", func() {
+			Eventually(cli.AppList, "30s", "5s").Should(ContainSubstring(appName))
+		})
+
+		It("app show includes domain name", func() {
+			appShowOutput, err := cli.AppShow(appName)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(appShowOutput.Name).To(Equal(appName))
+			Expect(appShowOutput.URI).To(Equal(domainName))
+		})
+	})
+
+	Context("when adding new environment", func() {
+		var err error
+		BeforeAll(func() {
+			_, err = cli.EnvInit(&client.EnvInitRequest{
+				AppName: appName,
+				EnvName: "test",
```

Comment:
i'm probably dumb but how does cloudfront get enabled 🥺

Reply:
+1, I was also searching for manifest or code that adds enable cdn snippet to the env manifest.

Reply:
Oh I forgot to include the manifest files